Detectors/MUON/MID/Calibration/macros/build_rejectlist.C

```diff
@@ -316,25 +316,53 @@ RejectListStruct load_from_json(const o2::ccdb::CcdbApi& ccdbApi, const char* fi
 {
   // Open the JSON file
   std::cout << "Reading reject list from file " << filename << std::endl;
-  RejectListStruct rl;
   std::ifstream inFile(filename);
   if (!inFile.is_open()) {
     std::cerr << "Could not open the file!" << std::endl;
-    return rl;
+    return {};
   }
 
   // Create an IStreamWrapper for file input stream
   rapidjson::IStreamWrapper isw(inFile);
-
   rapidjson::Document doc;
   if (doc.ParseStream(isw).HasParseError()) {
     std::cerr << "Problem parsing " << filename << std::endl;
-    return rl;
+    return {};
   }
-  auto startRange = o2::ccdb::BasicCCDBManager::getRunDuration(ccdbApi, doc["startRun"].GetInt());
-  auto endRange = o2::ccdb::BasicCCDBManager::getRunDuration(ccdbApi, doc["endRun"].GetInt());
-  rl.start = startRange.first;
-  rl.end = endRange.second;
+  
+  // manual-validity interval in ms:
+  int64_t startTSms = 0;
+  int64_t endTSms = 0;
+
+  // check if there are non-zero timestamps in the json
+  bool hasStartTT = doc.HasMember("startTT") && doc["startTT"].IsInt64() && doc["startTT"].GetInt64() != 0;
+  bool hasEndTT   = doc.HasMember("endTT")   && doc["endTT"].IsInt64()   && doc["endTT"].GetInt64()   != 0;
+  if (hasStartTT && hasEndTT) {
+    startTSms = doc["startTT"].GetInt64();
+    endTSms   = doc["endTT"].GetInt64();
+
+    // sanity check against the run boundaries
+    int   startRun = doc["startRun"].GetInt();
```

Comment:
Since you need the startRun and endRun anyways you can get it once outside the if